packages/react/src/primitives/Rating/__tests__/RatingIcon.test.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { RatingIcon } from '../RatingIcon';
+import { ComponentClassName } from '@aws-amplify/ui';
+
+const CustomIcon = ({ className }: { className: string }) => (
+  <svg className={className}></svg>
+);
+
+describe('RatingIcon', () => {
+  it('should render rating icon class', async () => {
```

Comment:
Is `async` necessary?

Reply:
No it does not. Just to be clear, this is because the test does not use the await operator?

Reply:
Yes, `await` must be used in an async function